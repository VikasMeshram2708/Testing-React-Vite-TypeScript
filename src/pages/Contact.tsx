export default function Contact() {
    return (
      <section data-testId="contactPage">
        <h1>Contact Page</h1>
      </section>
    );
  }
  