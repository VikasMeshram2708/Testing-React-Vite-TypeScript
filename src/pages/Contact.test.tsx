import { render, screen } from "@testing-library/react";
import Contact from "./Contact";

describe("testing contact page", () => {
  it("check the heading", () => {
    render(<Contact />);
    
    const ContactPage  = screen.getByTestId("contactPage");
    expect(ContactPage).toMatchSnapshot();

  });
});
