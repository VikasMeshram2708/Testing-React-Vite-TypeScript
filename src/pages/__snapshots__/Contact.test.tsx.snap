// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing contact page check the heading 1`] = `
<section
  data-testid="contactPage"
>
  <h1>
    Contact Page
  </h1>
</section>
`;
